views/default/page/layouts/elements/filter.php

```diff
@@ -1,22 +1,36 @@
 <?php
-
 /**
  * Layout content filter
  *
- * @uses $vars['filter']       An optional array of filter tabs
- *                             Array items should be suitable for usage with
- *                             elgg_register_menu_item()
+ * @uses $vars['filter']       - false or '' for no filter
+ *                             - a string for self defined filter content
```

Comment:
This broke my filters. In BC filters, it was possible to set the custom filter view. In 3.x logic, a string was meant to signify the filter type, which was then used as the handle for the filter menu hook.

Reply:
Never mind. I should have used filter_id